prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -179,35 +176,51 @@ class End2EndUniversalPipeline : CoolTest() {
                     bad("***async end2end_up FAILED***: Translate result invalid")
                 }
 
-                // check batch step
-                val translateReportId = getSingleChildReportId(routeReportId)
-                    ?: return bad(
-                        "***async end2end_up FAILED***:" +
-                            " Did not find a translate report id from route: $routeReportId"
-                    )
-
-                val batchResults = pollForStepResult(translateReportId, TaskAction.batch)
-                if (batchResults.isEmpty()) {
-                    return bad(
-                        "***async end2end_up FAILED***: No batch report was found after translate: $translateReportId"
-                    )
-                }
-                // verify each result is valid
-                for (result in batchResults.values)
-                    passed = passed && examineStepResponse(result, "batch", sender.topic)
-                if (!passed) {
-                    bad("***async end2end_up FAILED***: Batch result invalid")
-                }
+                if (sender.topic.isSendOriginal) { // original message pass through has translate directly call send
```

Comment:
Since topic properties now effect lineage, we need to consider the topic in this test that verifies lineage. This missing check (and the other one in TestReportStream) was the reason these tests started failing for the elims-elr topic when we updated it to use the pass through functionality.